tests/test_admin.py

```diff
@@ -1985,6 +1985,22 @@ def test_missing_grouper(self):
             response, "/en/admin/djangocms_versioning/pollcontentversion/?e=1"
         )
 
+    def test_missing_grouper_does_not_exist(self):
+        """Go to changelist with a grouper that does not exist in querystring
```

Comment:
Was this functionality of redirecting specified by Kuldeep?

Reply:
It was not, but from a UX stand point it made sense. I know, I know I've been wrong before.